### PR TITLE
Allow reading configuration information from file

### PR DESCRIPTION
**Pull request by ryanwcraig**
Allow reading in from a config.txt file, format is able to stay the same (can copy/paste the data from your source code to a file named config.txt, and you're good to go)

### CODE REVIEW

File: updater-v2.py [hunk 1]
```diff
@@ -7,9 +7,23 @@
...
-TOKEN = ''
-DOMAIN = ''
-RECORD = ''
+file_data = []
+
+try:
+    with open("config.txt","r") as f:
+        file_data = f.readlines()
+except:
+    print ("No config file found!")
```
Comment (ryanwcraig) [inline updater-v2.py]: Maybe allow an option for manual entry here.

**Comment by bensquire** [comment]
@ryanwcraig What format does the config.txt file take? Do the parameters need to be structured in a certain way?

**Comment by ryanwcraig** [comment]
Sorry for closing/reopening - I thought I'd have to open a new pull request for the changes I just made with exception handling. 

The format is exactly the same as in the script, so you can copy/paste from the Python script right into the config.txt file. For example,

TOKEN = 'TOKENHERE'
DOMAIN = 'example.com'
RECORD = 'record'

**Comment by bensquire** [comment]
Ah yes, I see it. We don't have unit tests (and I don't python), so are you happy this is functional enough to merge?

**Comment by ryanwcraig** [comment]
I've tested it and it's functional with both conditions (file exists and file doesn't exist/error), so I'm happy that it's functional to merge. The only problem I see is if that config file doesn't exist, it's going to ask for the config information on every run, which could mess with it if it's running in cron/LaunchControl. 

It might be best if I work the functionality in to write the config info to a file first, now that i think about it.

**Comment by bensquire** [comment]
Thats good thinking, either that or have the script just halt as it's designed to run in a cron or similar...

**Comment by ryanwcraig** [comment]
Halt for user input, or halt as in exit?

**Comment by bensquire** [comment]
As in exit... unless users tell us that they are using it interactively, I'd just assume its something thats being ran periodically :)

**Comment by ryanwcraig** [comment]
That makes sense! I'll have it throw an exception, I think that's justified in this case.

**Comment by bensquire** [comment]
Excellent, thank you!

**Comment by ryanwcraig** [comment]
No problem! I'll have that updated shortly.

**Comment by ryanwcraig** [comment]
There we go, tested and ready to merge. It raises a SystemExit when there's no config file available, or that file is not properly formatted.

**Comment by bensquire** [comment]
Awesome thanks!

**Pull request merged**